sgt-back-end/errorCheck.js

```diff
@@ -0,0 +1,33 @@
+const { jsonError } = require('./status');
+
+const errorCheck = (gradeId, name, course, score, req, res) => {
```

Comment:
You don't use `req`. It's tempting to have it there to make this feel more like an express handler function, but I'd probably take it off (and fix all the call sites). When I see it being passed in below, I wonder what `errorCheck` is going to do with it and why the extractions I just did weren't good enough.